sections/futureWork.tex

```diff
@@ -0,0 +1,8 @@
+\subsection{Future Work}\label{sec:future_work}
+
+The method, we have presented, can be expanded in different ways. 
+An interesting approach, could be to investigate other versions of \gls{lda}, such as the Dynamic Topic Model\cite{blei2006dynamic}, which is good at finding specific topics over time, which might improve the retrieval of old and important articles within a large dataset.
+There is also the approach of \cite{blei2007correlated}, where the correlation of topics is taken into account, which might yield improved information retrieval performance.
```

Comment:
```suggestion
There is also the approach of \citeauthor{blei2007correlated}\cite{blei2007correlated}, where the similarity between topics are taken into account.
This could make it easier to find appropriate hyperparameter settings as it makes it possible to evaluate how distinct topics in a topic model are.
```